### DISCUSSION TITLE
I want  import bootstrap at app.vue. But an error occurred

### DISCUSSION THREAD

**Discussion by JangCool** [nuxt/nuxt#18294] [Questions]
Let me ask you a few questions.

1. I think pages/index.vue will do the same function. Should I make the app.vue file mandatory? 

2. I want  import bootstrap at app.vue. But an error occurred.  How do I import a library managed in package.json?
```
<template>
  <div>app<br>
    <!-- Remove this component to get started! -->
    <NuxtPage />
  </div>
</template>
<script>

import '@/assets/scss/hamonica.scss';
import 'bootstrap'

export default {
    
}
</script>
```

![image](https://user-images.githubusercontent.com/2518021/138012206-6eb95a72-f670-41a3-9008-0fb5e7ba9d53.png)

![image](https://user-images.githubusercontent.com/2518021/138012124-ad486b6d-d15d-4286-b1e7-da2d388f31f3.png)

**Comment by wobsoriano** [comment]
Try this

```ts
// nuxt.config.ts
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
    css: [
        '~/assets/style.scss'
    ],
    meta: {
        script: [
            'bootstrap/dist/js/bootstrap.bundle.min.js'
            // { src: 'bootstrap/dist/js/bootstrap.bundle.min.js', body: true }, if you want to append before </body>
        ]
    }
})
```

```scss
// assets/style.scss
@import "../node_modules/bootstrap/scss/bootstrap"
```

**Reply by JangCool** [reply]
@wobsoriano um... It's my first time using nuxt and ask questions because I don't understand the nuxt .

Do I have to set it in the same way when using ``` yarn add or npm install <jslib> ``` ?

It's a little cumbersome. Is there an easy way to apply it?

**Reply by wobsoriano** [reply]
1. `npm install bootstrap`
2. Create `assets/style.scss` file and add the code above
3. Open `nuxt.config.ts` and add the code above
